spec.md

```diff
@@ -0,0 +1,121 @@
+# Specification
+
+Implement in your favorite languages by following this specification.
+
+* Date: 2015-09-07
+* Authors: Hsiaoming Yang, Jiangge Zhang
+
+
+## Interface
+
+Namespace should always be **GB2260**.
+
+## Data Structure
+
+There is a standard structure named `Division`. Those following properties
+should be included in it.
+
+- `code`: (str) The six-digit number of the specific administrative division.
+- `name`: (str) The Chinese name of the specific administrative division.
+- `year`: (str) Optional. The revision year, and empty means "latest".
```

Comment:
> 1. Those following properties **should** be included in it.
> 2. - `year`: (str) **Optional.** The revision year, and empty means "latest".

I think the bolded words have conflicts, and not describing `null` situation.

I suggest change **should** to **must**, then remove **Optional.**, and change **empty** to `null`.

That is:

> 1. Those following properties must be included in it.
> 2. - `year`: (str|null) The revision year, and `null` means "latest".